org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/util/date/DateUtility.java

```diff
@@ -70,14 +70,33 @@ public static String formatTime(Date d) {
   }
 
   /**
-   * format time with {@value DateFormat#SHORT}, {@value DateFormat#SHORT} patterns
+   * format time with {@value DateFormat#DEFAULT}, {@value DateFormat#SHORT} patterns
```

Comment:
I know it was already there but please improve JavaDoc:

Write the first letter in uppercase and use third person. Also use @link instead of @value.

Proposal: Formats date with the {@link DateFormat#DEFAULT} pattern and time with {@link DateFormat#SHORT}.